str/trtools/qc_str_runner.py

```diff
@@ -0,0 +1,122 @@
+#!/usr/bin/env python3
+# pylint: disable=duplicate-code
+"""
+This script runs qcSTR() from TRTools package on a single/merged STR vcf file and outputs various QC graphs
+
+For example: 
+analysis-runner --access-level test --dataset tob-wgs --description 'tester' --output-dir 'tester' qcSTR_runner.py --caller=eh --file-path=gs://cpg-tob-wgs-test/hoptan-str/mergeSTR/mergeSTR_2_samples_gangstr.vcf
+--refbias-binsize=5 --refbias-metric=mean --refbias-mingts=100 --refbias-xrange-min=0 --rebias-xrange-max=10000
+
+Required packages: sample-metadata, hail, click, os
+pip install sample-metadata hail click
+"""
+
+import click
+
+from cpg_utils.config import get_config
+from cpg_utils.hail_batch import output_path
+from cpg_workflows.batch import get_batch
+
+config = get_config()
+
+TRTOOLS_IMAGE = config['images']['trtools']
+
+
+# inputs:
+# file-path
+@click.option('--file-path', help='gs://...')
+# caller
+@click.option(
+    '--caller',
+    help='gangstr or eh',
+    type=click.Choice(['eh', 'gangstr'], case_sensitive=True),
+)
+# reference bias plot options
+@click.option(
+    '--refbias-binsize',
+    help=' Sets the binsize (in bp) used to bin x-axis values, which give the reference TR length. Default=5',
+)
+@click.option(
+    '--refbias-metric',
+    help='Determines which metric to use to summarize the reference bias in each bin. Default=mean. Must be one of: mean or median.',
+)
+@click.option(
+    '--refbias-mingts',
+    help='Exclude points computed using fewer than this many genotypes. This option is meant to avoid plotting outlier points driven by bins with small numbers of TRs with that reference length. Default=100',
+)
+@click.option(
+    '--refbias-xrange-min',
+    help='Exclude points corresponding to TRs with reference length less than this value. Default = 0',
+)
+@click.option(
+    '--refbias-xrange-max',
+    help='Exclude points corresponding to TRs with reference length greater than this value. Default = 10000',
+)
```

Comment:
```suggestion
@click.option(
    '--refbias-binsize',
    help=' Sets the binsize (in bp) used to bin x-axis values, which give the reference TR length', default=5, type=int
)
@click.option(
    '--refbias-metric',
    help='Determines which metric to use to summarize the reference bias in each bin.', type=click.Choice(['mean', 'median'], case_sensitive=True), default='mean'
)
@click.option(
    '--refbias-mingts',
    help='Exclude points computed using fewer than this many genotypes. This option is meant to avoid plotting outlier points driven by bins with small numbers of TRs with that reference length.', type=int, default=100
)
@click.option(
    '--refbias-xrange-min',
    help='Exclude points corresponding to TRs with reference length less than this value.', type=int, default=0
)
@click.option(
    '--refbias-xrange-max',
    help='Exclude points corresponding to TRs with reference length greater than this value.', type=int, default=10000
)
```

Instead of just describing the options you can include default values, meaning that you won't need to use every flag when you're running the script. This also has the benefit of being self-describing (type, default)